ethereum/clrfund
# Cart info in tallying phase incorrect

Fixes: https://github.com/ethereum/clrfund/issues/104

Testing cases:
- Make sure tallying phase still shows correct committed items
- Remove all cart items in reallocation, make sure tallying phase still shows correct committed items
- Edit cart but dont commit while in reallocation, make sure tallying phase still shows correct committed items
- Edit cart in reallocation phase, make sure tallying phase still shows correct committed items
- Refresh app, make sure tallying phase still shows correct committed items
